### PR TITLE
event_tracking

### PR DESCRIPTION
**Pull request by thasaarah**

### CODE REVIEW

**Comment by Ipuch** [comment]
Almost mergeable, I would say ?

File: event_tracking/constraint_task_quat_muscles.py [hunk 1]
```diff
@@ -0,0 +1,348 @@
+import biorbd_casadi as biorbd
```
Comment (Ipuch) [inline event_tracking/constraint_task_quat_muscles.py:1]: """
Did not converged.
"""

**Review by Ipuch — COMMENTED** [review]
some little things

**Comment by thasaarah** [comment]
Ready to merge

**Comment by Ipuch** [comment]
Congratulations 🎉

**Pull request merged**